portal/src/graphql/portal/OnboardingFormContainer.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback } from "react";
```

Comment:
Since the `FormContainer` and `OnboardingFormContainer` have different layout, `FormContainer` have the save and reset bottom on the top bar, and `OnboardingFormContainer` need to have the button at the bottom. My first thought is I don't want to make `FormContainer` supports multiple layout as we may have more layouts in future and it will make `FormContainer` difficult to maintain. Thought?